app/controllers/crops_controller.rb

```diff
@@ -52,29 +38,20 @@ def wrangle
     @crops = @crops.paginate(page: params[:page])
 
     @crop_wranglers = Role.crop_wranglers
-    respond_to do |format|
-      format.html
-    end
   end
 
   # GET /crops/hierarchy
   def hierarchy
     @crops = Crop.toplevel
-    respond_to do |format|
-      format.html
-    end
   end
 
   # GET /crops/search
   def search
     @term = params[:term]
-    @matches = Crop.search(@term)
+    @matches = Crop.approved.search(@term)
```

Comment:
Note: this is a functionality change... Maybe we wanted crops that are not approved to be findable?